### ISSUE TITLE
All filter  does't effect by  Use Everything filters

### ISSUE THREAD

**Issue by bestv5** [srwi/EverythingToolbar#498]
### Preflight Checklist

- [X] I have fully read the [README](https://github.com/srwi/EverythingToolbar/blob/master/README.md) and it did not solve the problem.
- [X] I have searched all [open and closed issues](https://github.com/srwi/EverythingToolbar/issues?q=) for the same bug report without success.

### EverythingToolbar Version

1.3.3.0

### Everything Version

Everything-1.4.1.1024.x64.zip

### Windows Version

win10

### Steps to reproduce

1. edit Everything  filters (all),add search rule  "(?<!.class)$"  .
2. select  'Use Everything filters' in Everythingtoolbar.
3. in Everythingtoolbar, all does not effect  like in Everything.

### Expected Behavior

same as search in Everything

### Actual Behavior

 .class file show in filter(all).

### Log output

not file

**Comment by srwi** [comment]
Hi, the problem is resolved and the fix will be available in the next release!

**Comment by RealSubstantiality** [comment]
> Hi, the problem is resolved and the fix will be available in the next release!

Hello there! It does seem that the "RegEx not working in filter" problem in this thread is fixed. However, it seems that the RegEx is not working in "Rules". 
I added three custom commands with different RegEx. I tested them all by searching in Everything and they work as intended, showing only files I want for each expression. However, all three rules will appear in the "open with..." menu for every file or folder. Even commands limited to files can appear for folders, which is not relevant with RegEx.
I am a new user and I don't know if I did this the right way. I searched similar issues with no success. Sorry if I interrupted!
